### DISCUSSION TITLE
Welcome to termvisage Discussions!

### DISCUSSION THREAD

**Discussion by AnonymouX47** [AnonymouX47/termvisage#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by krw6711** [comment]
Bro, I have a suggestion.
Why not make it a library can be imported in Python code and compiled with it and uses these good projects in.
I am so happy to find this helpful stuff ☺️ I searched a lot for something that can do that. I use Arch, btw

**Reply by AnonymouX47** [reply]
Hello!

Thanks for the suggestion and compliments. :smiley:

Anyways, it actually exists already (as [term-image](https://github.com/AnonymouX47/term-image)) and is what this project is built upon.

The library is actually mentioned (and its repository linked to) a couple times within this project's README and the repository's description. I'll consider making it more obvious though, for the sake of others.

Thanks once again.
